### ISSUE TITLE
Failing/disabled python API BatchNormalization test

### ISSUE THREAD

**Issue by mdabros** [Microsoft/CNTK#3168]
Hi,

Currently the BatchNormalization test for the Python API is disabled:
[batch-norm-test](https://github.com/Microsoft/CNTK/blob/master/bindings/python/cntk/layers/tests/layers_test.py#976)

Running the test shows that expected should be:
`[1.9999199 -1.9999199  1.9999199 -1.9999199]`

But the actual result is:
`[632.45557 316.22778 632.45557 316.22778]`

Does this mean that the current implementation of BatchNormalization has an error, and what implications does this have on using BatchNormalization when training models using CNTK with Python? 

Will the test be corrected and enabled at some point?

best regards
Mads

**Comment by KeDengMS** [comment]
The expected value in this test is wrong. During eval, the run_mean/run_variance is not computed from the input, so they would be at their init value of 0, thus the compute becomes:
```
(x - 0)/sqrt(0 + epsilon) * scale + bias
```
Because the default epsilon is 1e-5, scale=2, bias=0, the expected output should be match the output.
